lib/tasks/add_countries.rake

```diff
@@ -0,0 +1,37 @@
+namespace :add_countries do
+  desc "Add countries to support interface"
+  task generate: :environment do
```

Comment:
I suggest we name the Rake task specific to this one time, since we're going to run it once and then never again. Something like `countries:create_feb_1` perhaps?